### PR TITLE
Add Python package `ndbc-api` from PyPi

### PR DESCRIPTION
**Pull request by CDJellen**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by CDJellen** [comment]
Thank you in advance for your time and review; I maintain the `ndbc-api` package [here on GitHub](https://github.com/CDJellen/ndbc-api) and [here on PyPi](https://pypi.org/project/ndbc-api/); I recently had [a user request that I make it available through conda](https://github.com/CDJellen/ndbc-api/issues/29) and am willing to maintain the conda distribution in addition to the PyPi distribution.

**Comment by CDJellen** [comment]
@conda-forge/help-python, ready for review!

Thank you in advance for your assistance and please let me know if there are any required adjustments;

**Comment by CDJellen** [comment]
@conda-forge/help-python please let me know if there is anything else I can do to assist in this PR;

**Comment by CDJellen** [comment]
@conda-forge-admin, please ping team

Thanks again, please let me know how to assist in getting this PR merged;

**Comment by CDJellen** [comment]
@conda-forge/help-python

The initial request might not have been logged with successful runs; I apologize for any inconvenience;

File: recipes/ndbc-api/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{% set name = "ndbc-api" %}
+{% set version = "0.24.6.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/45/e5/b617b383785760e3d699ee81f37f4ad253fb9fa969084949db995090b3ce/ndbc_api-0.24.6.12.1.tar.gz
```
Comment (ocefpaf) [inline recipes/ndbc-api/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ndbc_api-{{ version }}.tar.gz
```

**Comment by CDJellen** [comment]
Thank you so much for the suggestions @ocefpaf ! I've made the requested changes and test are passing; please let me know if there is anything else I can to to help and thanks again for your time.

**Pull request merged**